t/custom-formats.t

```diff
@@ -0,0 +1,45 @@
+use lib '.';
+use t::Helper;
+use Mojo::Util 'decode';
+use Test::More;
+
+my $schema = {type => 'object', properties => {v => {type => 'string'}}};
+
+{
+  local $schema->{properties}{v}{format} = 'palindrome';
```

Comment:
Instead of using 'local' to patch a single schema, I would just create a new document here, which would make it easier to see what it looks like as a whole.

Reply:
I think this is probably written this way, since I've done that in other tests. Not saying it's the best way, but it saves a lot of keystrokes.

Reply:
Sorry, past month was very stressing. I have just followed your recommendation @karenetheridge  in https://github.com/fairtracks/json-validator/commit/94cba172282bb1610c250413d01c5a651dd3a12c